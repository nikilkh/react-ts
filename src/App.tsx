import StateContextProvider from "./components/context";
import { LandPage } from "./components/LandPage";




export default function App() {

  return (
    <>
    <StateContextProvider>
      <LandPage></LandPage>
    </StateContextProvider>
   </>
  );
}
