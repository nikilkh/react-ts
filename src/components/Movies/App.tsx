import { Route, Routes } from "react-router-dom";
import BookingPage from "../Tickets/BookingPage";
import { Main } from "./Main";

export const App = () => {
  return (
    <Routes>
      <Route path="/" element={<Main />} />

      <Route
        path="/book/:id/:title"
        element={<BookingPage children={undefined} />}
      ></Route>
    </Routes>
  );
};
