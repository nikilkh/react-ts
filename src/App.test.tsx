import { render } from "@testing-library/react";
import { App } from "./App";
import { BrowserRouter } from "react-router-dom";

describe("<App/>", () => {
  it("should take snapshot", () => {
    const { container } = render(
      <BrowserRouter>
        <App />
      </BrowserRouter>
    );
    expect(container).toMatchSnapshot();
  });
});
