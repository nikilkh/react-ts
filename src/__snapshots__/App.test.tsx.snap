// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App/> should take snapshot 1`] = `
<div>
  <div>
    <div
      class="sc-hAZoDl joektr"
    >
      <header
        class="sc-fnykZs csVgBM"
      >
        <h1>
          Movie List
        </h1>
      </header>
      <div
        class="sc-fEOsli cmNnSW"
      >
        <input
          placeholder="search your movie here"
          value=""
        />
        <button
          class="sc-bjUoiL unMHp"
          data-testid="searchButton"
        >
          <img
            alt="search icon"
            src="searchIcon.svg"
          />
        </button>
      </div>
      <div
        class="sc-idiyUo kUDrug"
      >
        <h3>
          loading...
        </h3>
      </div>
    </div>
  </div>
</div>
`;
